comp/otelcol/otlp/components/exporter/datadogexporter/metadata.go

```diff
@@ -0,0 +1,34 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+package datadogexporter
+
+import (
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/otel/metric"
+	"go.opentelemetry.io/otel/trace"
+)
+
+// Type is the type of the exporter
+var Type = component.MustNewType("datadog")
+
+const (
+	// LogsStability is the stability level of the logs datatype.
+	LogsStability = component.StabilityLevelBeta
```

Comment:
This is alpha per https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/datadogexporter#datadog-exporter

Reply:
I think it should be beta in OTel Agent. in DD exporter we were waiting to use logs agent pipeline to mark it beta